app/views/admin/works/_combined_audio_derivatives.html.erb

```diff
@@ -0,0 +1,53 @@
+<div class="card bg-light mb-3">
+  <h2 class="card-header h3">Combined Audio Derivative Files</h2>
+  <div class="card-body">
+    <% if view.work_audio_members.count == 0 %>
+      This oral history doesn't have any audio files associated with it.
+    <% else %>
+      <% if view.combined_audio_fingerprint.present? %>
+        <% if CombinedAudioDerivativeCreator.new(model).fingerprint != view.combined_audio_fingerprint %>
```

Comment:
You *could* extract this logic into the ViewModel, once you have a ViewModel it's best to keep logic in the html template as simple as possible, mostly just calling methods in the viewmodel for anything non-trivial. 

But I'm not gonna block the PR over it at this point!

Reply:
Good call; done.